extensions/functions_arithmetic.yaml

```diff
@@ -276,3 +276,105 @@ aggregate_functions:
         decomposable: MANY
         intermediate: "STRUCT<fp64,i64>"
         return: fp64?
+  - name: "min"
+    description: Min a set of values.
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
```

Comment:
I don't see the purpose of multiple overflow behaviors for min. How would min ever overflow?

Reply:
I just got the meaning of the `options` which indicates the behavior of a function overflow. So will remove it.

Reply:
done